tests/pos/13990.scala

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
usual file convention is `i13990.scala`

Reply:
plenty of files in this folder don't follow this convention :p
But yes, you're right, a letter is missing here (less dramatic than in "La Disparition" but still)